api/v1alpha1/nats_types.go

```diff
@@ -75,6 +77,7 @@ undefined they will be interpreted as "" and result in 0 instead of being replac
 
 // NATS is the Schema for the NATS API.
 // +kubebuilder:object:root=true
+// +kubebuilder:resource:path=nats
```

Comment:
The new version of `controller-gen` was setting the plural as `natses` instead of `nats.